pages/products/product.css

```diff
@@ -53,15 +50,16 @@
 }
 /* product cards display */
 .product-cards-display{
-    /* display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-content: space-between;
-    align-items: flex-start; */
+    margin-top: -670px;
     width: 70%;
     padding: 16px;
-    /* display: flex; */
-    /* justify-content: space-evenly; */
-    /* flex-wrap: wrap; */
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
     float: right;
```

Comment:
Any specific reason to use floats here? Could have used margin-right : auto maybe. Let me now your thoughts about this?

Reply:
not a specific reason. just wanted product cards to right of the sidebar. now i changed to margin-left:auto it s giving same result thanks